README.md

```diff
@@ -1,7 +1,75 @@
-# android-sdk
-[WIP] One merchant integration point for all of PayPal's services
+# PayPal Android SDK
+Welcome to the PayPal Android SDK. This library will help you accept card, PayPal, Venmo, and alternative payments in your Android app.
```

Comment:
```suggestion
Welcome to the PayPal Android SDK. This library will help you accept Card, PayPal, and Venmo payments in your Android app.
```

I could be wrong, but I think "Card" is a proper noun here since we're talking about a feature of the SDK. Also, we could omit alternative payments and explicitly list out future features as the SDK evolves.